### PR TITLE
Update schema.sql

### PR DESCRIPTION
**Pull request by glombers**
https://otland.net/threads/znote-tfs-1-2-error-when-login-to-account-need-help-secret-column-missing.249629/

OR This:
https://github.com/Znote/ZnoteAAC/pull/281

### CODE REVIEW

**Comment by DSpeichert** [comment]
This column is already created in TFS 1.2 with a migration:
https://github.com/otland/forgottenserver/blob/1.2/data/migrations/18.lua

However the included schema.sql has version set to 18 but is missing this column.

Please see my comments

File: schema.sql [hunk 1]
```diff
@@ -7,6 +7,7 @@ CREATE TABLE IF NOT EXISTS `accounts` (
   `lastday` int(10) unsigned NOT NULL DEFAULT '0',
   `email` varchar(255) NOT NULL DEFAULT '',
   `creation` int(11) NOT NULL DEFAULT '0',
+	`secret` char(16) DEFAULT NULL,
```
Comment (DSpeichert) [inline schema.sql]: * There's unusual whitespace
* The column should be added after `password` to be consistent with https://github.com/otland/forgottenserver/blob/1.2/data/migrations/18.lua

Comment (glombers) [inline schema.sql]: Only a player does not turn on the server can not set up an account on znote aac

**Comment by DSpeichert** [comment]
It's a valid PR and can get merged if you just make the changes I asked for, no need to abandon.

**Comment by ranisalt** [comment]
It's not valid. You are using an awfully outdated commit, [this column has been added by Znote himself MONTHS ago](https://github.com/otland/forgottenserver/commit/7dcacb958f693db4a286ff17c22b35b97eea26f4).

**Comment by DSpeichert** [comment]
@ranisalt Znote's fix was committed to the master branch. This is a bugfix for the 1.2 branch. It's still broken in 1.2 right now.

**Comment by ranisalt** [comment]
Hmmm understood. You have write access to his PR, so you can fix it and merge :)

**Comment by DSpeichert** [comment]
I'm doing it now...

**Comment by Znote** [comment]
https://github.com/otland/forgottenserver/pull/1963

**Pull request closed without merging**